### PR TITLE
Add support for mcs client, management cvx and api models.

### PR DESCRIPTION
**Pull request by morinpa**
## Change Summary

Added support for the following 

management cvx
management api models
mcs client

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Documentation content changes
- [ ] Other (please describe):

## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes

added new jinja2 templates to the eos_cli_config_gen role. 
```
mcs-client.j2
management-cvx.j2
management-api-models.j2

```

## Checklist:

- [X] My code has been rebased from media-cli-fixes before I start
- [ ] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly
- [ ] All new and existing tests passed ([`pre-commit`](https://www.avd.sh/docs/installation/development/#python-virtual-environment), `make linting` and `make sanity-lint`).

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-gnmi.j2 [hunk 1]
```diff
@@ -13,7 +13,31 @@ management api gnmi
       vrf {{ vrf }}
 {%         endif %}
 {%     endfor %}
-{%     if management_api_gnmi.octa is arista.avd.defined %}
+{%     if management_api_gnmi.octa is defined %}
```
Comment (sundayN) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-gnmi.j2]: we should use `arista.avd.defined` because it also tests for `None` [defined-test](https://avd.sh/en/latest/plugins/index.html#defined-test)

Comment (morinpa) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-gnmi.j2]: fixed
